src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapSeparatorAndInterfaces.java

```diff
@@ -1,6 +1,6 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
```

Comment:
Look
InputOperatorWrap.java was moved to directory /operatorwrap by person, who was splitting tests for OperatorWrap but he/she didn't modify package in this file from com.puppycrawl.tools.checkstyle.checks.whitespace to com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap
So javac loocked in com.puppycrawl.tools.checkstyle.checks.whitespace and saw two classes InputOperatorWrap.java and InputSeparatorWrap.java. As in InputOperatorWrap we use inner declared interfaces in InputSeparatorWrap there. So as package wasn't modified to meet the requirements it wasn't arguing by compiler.